xml/System.Windows.Forms/SplitterPanel.xml

```diff
@@ -136,17 +129,10 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>Gets or sets a value indicating whether the <see cref="T:System.Windows.Forms.SplitterPanel" /> is automatically resized to display its entire contents. This property is not relevant to this class.</summary>
+        <summary>Gets or sets a value that indicates whether the <see cref="T:System.Windows.Forms.SplitterPanel" /> is automatically resized to display its entire contents. This property is not relevant to this class.</summary>
         <value>
           <see langword="true" /> if the <see cref="T:System.Windows.Forms.SplitterPanel" /> is automatically resized; otherwise, <see langword="false" />.</value>
-        <remarks>
-          <format type="text/markdown"><![CDATA[  
-  
-## Remarks  
- This property is not relevant to this class.  
-  
- ]]></format>
-        </remarks>
+        <remarks>To be added.</remarks>
```

Comment:
Is `To be added` a known string that is not getting displayed on the docs website and in intellisense? I wonder why you are adding it instead of leaving the `remarks` tag empty?

Reply:
`To be added.` is the expected default string for elements we don't want to document.

Reply:
And correct, the website and intellisense won't display "To be added". Also, Remarks don't show up in intellisense, only in the website.